### DISCUSSION TITLE
ISCC-NBS Color Designations

### DISCUSSION THREAD

**Discussion by Ron024** [colour-science/colour#1230] [General]
I am working on porting a Matlab [implementation ](https://www.munsellcolourscienceforpainters.com/MunsellAndKubelkaMunkToolbox/MunsellAndKubelkaMunkToolbox.html) by Paul Centore of the [ISCC-NBS ](https://en.wikipedia.org/wiki/ISCC%E2%80%93NBS_system) system to python.  Would including this be in the scope of colour-science?

**Comment by thomasmansencal** [comment]
Hi @Ron024,

That would certainly be in the scope but we would need to get agreement from Paul to include his GPL 3 code under the BSD 3 license terms as we did for the Munsell one.

Cheers,

Thomas
